src/util/fabric/theme.ts

```diff
@@ -13,6 +13,12 @@ const fontStyles: IRawStyle = {
 
 // NOTE: Would be handy to have access to our CSS variable values in JS...
 loadTheme({
+  palette: {
+    themePrimary: '#6c98d9',
+    neutralSecondary: '#646d7a',
+    neutralTertiary: '#dde0e6',
+    neutralLighter: '#f3f5f8',
```

Comment:
Setting colors here let me overcome some challenges with specificity of css selectors. However, I think we need a better way to managed these values.  When @unindented and I talked, we thought it might be ideal to declare our named colors in a json file, and allow that to be included here. This json file would then also replace colors.css. Maybe as part of the build we could generate colors.css from colors.json.

Reply:
👍